Spoony-iOS/Spoony-iOS/Source/Feature/Agree/AgreeView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  AgreeView.swift
+//  Spoony-iOS
+//
+//  Created by 최주리 on 3/29/25.
+//
+
+import SwiftUI
+
+import ComposableArchitecture
+
+struct AgreeView: View {
+    @EnvironmentObject private var navigationManager: AuthNavigationManager
```

Comment:
환경변수말고

```suggestion
init(store: StoreOf< AgreeFeature>, navigationManager: AuthNavigationManager) {
    self.store = store
    self.navigationManager = navigationManager
}
```
의존성으로 전달해주면 좀더 이쁠거같아요

Reply:
AgreeView가 AuthNavigationManager 내부의 build 함수 안에서 생성되고 있어서 init으로 전달하는 방식을 사용하지 못했습니다 ! 혹시 방법이 있을까요 .....??